src/components/SignerPanel/SigningStatus.js

```diff
@@ -0,0 +1,290 @@
+
+    return {
+      iconColor: theme.hint,
+      label: 'Waiting for signature...',
+    }
+  }, [theme, status])
+
+  return (
+    <div
+      css={`
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          width: 60px;
+          height: 60px;
+          border: 2px solid ${iconColor};
+          border-radius: 50%;
+          transition: border-color 150ms ease-in-out;
+        `}
+      >
+        {status === TRANSACTION_STATUS_SIGN_FAILED ? (
+          <IconCross
+            size="medium"
+            css={`
+              color: ${iconColor};
+              transition: color 150ms ease-in-out;
+            `}
+          />
+        ) : (
+          <IconCheck
+            size="medium"
+            css={`
+              color: ${iconColor};
+              transition: color 150ms ease-in-out;
+            `}
+          />
+        )}
+      </div>
+      <p
+        css={`
+          margin-top: ${3 * GU}px;
+        `}
+      >
+        {label}
+      </p>
+    </div>
+  )
+}
+
+function SigningStatusInfo({
+  allSuccess,
+  isMultiTx,
+  maxAttemptsReached,
+  onClosePanel,
+  onNextAttempt,
+  transactions,
+  transactionsStatus,
+}) {
+  const { activated } = useWallet()
+  const provider = getProviderFromUseWalletId(activated)
+
+  if (allSuccess) {
+    return (
+      <Info>Transaction{isMultiTx ? 's' : ''} submitted successfully!</Info>
+    )
+  }
+
+  const requiresMultipleSignatures =
+    isMultiTx && transactions.filter(tx => !tx.skipSignature).length > 1
+  const transactionFailedStatus = transactionsStatus.find(status =>
+    [TRANSACTION_STATUS_SIGN_FAILED, TRANSACTION_STATUS_FAILED].includes(status)
+  )
+
+  if (transactionFailedStatus) {
+    if (isMultiTx) {
+      return (
+        <TransactionReattempt
+          maxAttemptsReached={maxAttemptsReached}
+          onClosePanel={onClosePanel}
+          onNextAttempt={onNextAttempt}
+        />
+      )
+    }
+
+    return (
+      <>
+        <Info>
+          {transactionFailedStatus === TRANSACTION_STATUS_SIGN_FAILED
+            ? "Your transaction wasn't signed"
+            : 'Your transaction failed'}
+        </Info>
+        <Button
+          css={`
+            margin-top: ${2 * GU}px;
+          `}
+          label="Close"
+          mode="strong"
+          onClick={onClosePanel}
+          wide
+        />
+      </>
+    )
+  }
+
+  if (isMultiTx) {
+    return (
+      <Info>
+        {requiresMultipleSignatures ? (
+          <span>
+            This action requires multiple transactions. Please sign them one
+            after another and <b>do not close this window</b> until the process
+            is finished. Open your Ethereum provider {provider.name} to sign the
+            transactions.{' '}
+            <Link href="#">I don’t see the Ethereum provider.</Link>
```

Comment:
I don't think we have a great place to link this to, and I'm not really even sure what we would say in it :(.

Maybe let's remove it for now?